com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/UserErrorException.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+* Copyright (c) 2017 Microsoft Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*
+* Contributors:
+*     Microsoft Corporation - initial API and implementation
+*******************************************************************************/
+
+
+package com.microsoft.java.debug.core;
+
+public class UserErrorException extends DebugException {
+    private static final long serialVersionUID = -6001456457602995764L;
```

Comment:
What is this magic number for? Please comment it

Reply:
serialVersionUID is generated by eclipse using random number, it is used serialization orelse there will be a compilation warning.